.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

Comment:
We should do weekly on all other packages as well, that is a good idea! I am sometimes annoyed when already next day dependabot complains ;)